Open Judge System/Web/OJS.Web/Areas/Contests/Views/Results/_SimplePartial.cshtml

```diff
@@ -0,0 +1,172 @@
+﻿@model ContestResultsViewModel
+
+@using OJS.Common.Models
+@using OJS.Web.Areas.Contests.Controllers
+@using OJS.Web.Areas.Contests.ViewModels.Results
+@using OJS.Web.Areas.Users.Controllers
+
+@using Resource = Resources.Areas.Contests.Views.ResultsSimple
+
+@{
+    const string ResultsContainerId = "simple-results-container";
+    const string LoadingMaskContainerId = "simple-results-loading-mask-container";
+    const string OnPageChangedEventHandlerName = "OnPageChanged";
+    var isOfficial = Model.IsCompete;
+    var results = Model.Results;
+}
+
+@section Styles {
+    <link href="/Content/Contests/results-page.css" rel="stylesheet" />
```

Comment:
Wait is this even working? i don't think you can use this section in a partial view.

Reply:
Done in https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/223